Hymns/Infrastructure/Data/PDFPreloader.swift

```diff
@@ -33,3 +39,11 @@ class PDFLoader {
         return cache[url]
     }
 }
+
+extension Resolver {
+    public static func registerPDFLoader() {
+        register(PDFLoader.self) {
+            return PDFLoaderImpl() as PDFLoader
```

Comment:
you don't really need the return statement if it's just one-liine

Reply:
Thanks!